keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -17,9 +17,74 @@
 
 import tensorflow as tf
 import tensorflow_text as tf_text
+from tensorflow import keras
 
 from keras_nlp.tokenizers import tokenizer
 
+# Pretrained Vocabularies
+
+BASE_PATH = "https://storage.googleapis.com/keras-nlp/pretrained-tokenizers/wordpiece-tokenizer/vocabularies/"
```

Comment:
It looks like on https://github.com/keras-team/keras-nlp/pull/299/files we are using underscores for file paths.

Let's also do `word_piece_tokenizer` not `wordpiece_tokenzier` to agree with the naming of this file. Also maybe delete the `vocabularies/` subdirectory? There is nothing else we need to store right?

Reply:
edited